### ISSUE TITLE
Python for Algorithm Trading - Chapter 5 - Page 158 - Typo

### ISSUE THREAD

**Issue by edgetrader** [yhilpisch/py4at#3]
### For recording purpose here

Code line 162

```
%%time
model.fit(training_data[cols], 
          training_data['direction'], 
          epochs=50, verbose=False, 
          validation_split=0.2, shuffle=False)
```

Should be to train model with normalised features

```
%%time
model.fit(training_data_[cols], 
          training_data['direction'], 
          epochs=50, verbose=False, 
          validation_split=0.2, shuffle=False)
```

[Dr Yves' has replied here](https://base.pqp.io/base/fo/get_topic?post_id=5379)

**Comment by BeneInfosec** [comment]
> ### For recording purpose here
> Code line 162
> 
> ```
> %%time
> model.fit(training_data[cols], 
>           training_data['direction'], 
>           epochs=50, verbose=False, 
>           validation_split=0.2, shuffle=False)
> ```
> 
> Should be to train model with normalised features
> 
> ```
> %%time
> model.fit(training_data_[cols], 
>           training_data['direction'], 
>           epochs=50, verbose=False, 
>           validation_split=0.2, shuffle=False)
> ```
> 
> [Dr Yves' has replied here](https://base.pqp.io/base/fo/get_topic?post_id=5379)

Yes, I think the same.
